### ISSUE TITLE
Sending message to a group

### ISSUE THREAD

**Issue by mohinderq** [orkestral/venom#742]
I've been using the venom bot for quite a while now. However for some reason I can't send messages to groups with new accounts.

I've created a new Whatsapp. Then created a whatsapp group. In the whatsapp group I say Hi!. The bot throws the error. 

```
  text: 'The number does not exist',
```

To send the message I am using:
```
await client.sendText(message.from, respo).then((res) => {console.log(res)}).catch((err) => {console.log(err)});
```

I've noticed using venom 3.0.8 doesn't have this problem.

**Comment by jonalan7** [comment]
are you using the most current version?

**Comment by jczimer** [comment]
We have the same problem. We are using the last version.

**Comment by mohinderq** [comment]
> are you using the most current version?

While using the most recent version I get this error. Downgrading to 3.0.8 seems to solve this problem.

**Comment by jczimer** [comment]
Fixed on version 3.0.11.. works to me
